packages/playground/_stories/main/TextArea/TextArea.stories.ts

```diff
@@ -23,29 +27,54 @@ export default {
     argTypes,
 } as Meta<TextArea>;
 
-const Template: UI5StoryArgs<TextArea, StoryArgsSlots> = (args) => html`<div></div>`;
+const Template: UI5StoryArgs<TextArea, StoryArgsSlots> = (args) => html`
+<ui5-textarea
+	id="textArea-${index++}"
+	value="${ifDefined(args.value)}"
+	?disabled="${ifDefined(args.disabled)}"
+	?readonly="${ifDefined(args.readonly)}"
+	?required="${ifDefined(args.required)}"
+	placeholder="${ifDefined(args.placeholder)}"
+	value-state="${ifDefined(args.valueState)}"
+	rows="${ifDefined(args.rows)}"
+	maxlength="${ifDefined(args.maxlength)}"
+	?show-exceeded-text="${ifDefined(args.showExceededText)}"
+	?growing="${ifDefined(args.growing)}"
+	growing-max-lines="${ifDefined(args.growingMaxLines)}"
+	name="${ifDefined(args.name)}"
+	accessible-name="${ifDefined(args.accessibleName)}"
+	accessible-name-ref="${ifDefined(args.accessibleNameRef)}"
+></ui5-textarea>`;
+
+export const BasicTextArea = Template.bind({});
+BasicTextArea.args = {
+	placeholder: 'Type as much text as you wish',
+};
+BasicTextArea.storyName = "Basic TextArea";
+
 
+export const TextAreaMaxLength = Template.bind({});
+TextAreaMaxLength.args = {
+	placeholder: 'Type no more than 10 symbols',
+	maxlength: 10,
+	showExceededText: true
 
-export const Template0: StoryFn = () => html`
-<h3>Basic TextArea</h3>
-	<div class="snippet">
-		<ui5-textarea class="textarea-width" placeholder="Type as much text as you wish"></ui5-textarea>
-	</div>
-`;
+};
+TextAreaMaxLength.storyName = "TextArea with Maximum Length";
 
+export const TextAreaLabel = Template.bind({});
+TextAreaLabel.decorators = [
+	(story) => {
+		return html`
+		<ui5-label for="textArea-${index}">Description</ui5-label>
+		${story()}
+		`;
+	}
+]
+TextAreaLabel.args = {
```

Comment:
here you can add storyName as the current one results in "Text Area Label", which is not consistent to the other story names for the textarea. Another option would be to correct the name of the other stories :)

Reply:
Done. I've corrected the names everywhere to have spaces - "Text Area ..."